scripts/environment_json.sh

```diff
@@ -41,74 +47,26 @@ while getopts rp:h? opt; do
 done
 shift $((OPTIND-1))
 
-function pull_from_config_xml() {
-  cat << EOM | node
-const fs = require('fs');
-const xml2js = require('xml2js');
-const s = fs.readFileSync('config.xml');
-xml2js.parseString(s, function(err, result) {
-  console.log($1);
-});
-EOM
-}
+if [[ "${TYPE}" == "release" && -z ${SENTRY_DSN:-} ]]; then
+  echo "SENTRY_DSN is undefined." 1>&2
```

Comment:
Let's be clearer about the failed condition.
```suggestion
  echo "Release builds require SENTRY_DSN, but it is not defined." 1>&2
```